### PR TITLE
Eclipsefunctions

### PR DESCRIPTION
**Pull request by vreijs**
Not 100% done yet (need to add the testthat tests for the new functions). 
But I also have an issues when doing roxygen2::roxygenise(). I get warning:
Warning: @details [C:\Users\vrhome\Documents\R\Projects\swephR\R\RcppExports.R#428]: mismatched braces or quotes

### CODE REVIEW

**Comment by vreijs** [comment]
will work on the testthat tests in coming days.

**Comment by rstub** [comment]
Nice! Is there anything missing from your side? If not, I will merge this PR. Before we can do a CRAN release, I have to figure out #52, though.

BTW, does this mean we now have full coverage of the SE C API?

**Comment by vreijs** [comment]
I still have this error when doing the roxygen2::roxygenise():
Warning: @details [C:\Users\vrhome\Documents\R\Projects\swephR\R\RcppExports.R#419]: mismatched braces or quotes

Indeed all SE C API are now covered.

File: R/RcppExports.R [hunk 1]
```diff
@@ -331,14 +415,26 @@ swe_azalt_rev <- function(jd_ut, coord_flag, geopos, xin) {
     .Call(`_swephR_azalt_rev`, jd_ut, coord_flag, geopos, xin)
 }
 
+#' @details
+#' \describe{
```
Comment (rstub) [inline R/RcppExports.R:419]: The warning comes from this line. Indeed, the brace opened on this line is never closed. However, this is a generated file ...

File: src/libswe/swecl.c [hunk 2]
```diff
@@ -3152,7 +3152,7 @@ static double calc_dip(double geoalt, double atpress, double attemp, double laps
 {
   /* below formula is based on A. Thom, Megalithic lunar observations, 1973 (page 32).
   * conversion to metric has been done by
-  * V. Reijs, 2000, http://www.iol.ie/~geniet/eng/refract.htm
```
Comment (rstub) [inline src/libswe/swecl.c:3155]: I will create e new patch file for this change

File: src/swephR.cpp [hunk 3]
```diff
@@ -611,14 +740,27 @@ Rcpp::List azalt_rev(double jd_ut, int coord_flag, Rcpp::NumericVector geopos, R
   return Rcpp::List::create(Rcpp::Named("xout") = xout);
 }
 
+//' @details
+//' \describe{
+//' \item{swe_refrac()}{Calculate either the topocentric altitude from the apparent altitude
+//' or the apparent altitude from the topocentric altitude.
+//' }
```
Comment (rstub) [inline src/swephR.cpp:747]: The brace opened on line 744 is never closed. This is the reason for the warning. It is probably best to add a closing brace in line 746, similar to other uses of this documentation pattern.

**Review by rstub — CHANGES_REQUESTED** [review]

**Comment by vreijs** [comment]
I have removed the warning of the brackets.

**Review by rstub — APPROVED** [review]

**Comment by vreijs** [comment]
Thanks!

**Pull request merged**